src/mattack_actors.cpp

```diff
@@ -66,7 +66,7 @@ bool leap_actor::call( monster &z ) const
 
     std::vector<tripoint> options;
     tripoint target = z.move_target();
-    float best_float = trig_dist( z.pos(), target );
+    float best_float = trigdist ? trig_dist( z.pos(), target ) : square_dist();
```

Comment:
```suggestion
    float best_float = trigdist ? trig_dist( z.pos(), target ) : square_dist( z.pos(), target );
```

Reply:
Why roll your own conditional instead of calling `rl_dist` here and in the other case below?  I guess because you want a float?  Maybe there should be a `rl_dist_float` function?

Reply:
Yea they are piling up, I'll take a look at just having rl_dist() return a float and it'll get cast to int most everywhere, that might work fine...